src/pages/saksbehandling/revurdering/OppsummeringPage/oppsummeringPageForms/OppsummeringPageForms.tsx

```diff
@@ -225,6 +235,8 @@ export const SendTilAttesteringForm = (props: {
                 ? props.revurdering.fritekstTilBrev
                 : tilbakekreving
                 ? formatMessage('tilbakekreving.forhåndstekst')
+                : erRevurderingOpphørPgaManglendeDokumentasjon(props.revurdering)
+                ? formatMessage('opplysningsplikt.forhåndstekst')
                 : '',
```

Comment:
Denne sjekken her ser ut å gå på rundgång nå på de forskjellige formsen, og er ikke serskilt pen eller lettlest. Kunde vi hatt noe standardopplegg for dette?